src/main/java/cs/ualberta/cmput402/tictactoe/board/ScoreBoard.java

```diff
@@ -0,0 +1,37 @@
+package cs.ualberta.cmput402.tictactoe.board;
+
+public class ScoreBoard {
+	
+	static private int[][] array;
+	
+	public ScoreBoard() {
+		array = new int[2][3];
+	}
+	
+	public void initScoreBoard() {
+		for(int i = 0; i < 2; i++) {
+			for(int j = 0; j < 3; j++) {
+				this.array[i][j] = 0;
+			}
+		}
+	}
+	
+	public void incWin(int idx) {
```

Comment:
Implementation of score is un-necessarily complicated. Please make functions more human-legible or add clear comments how the flow of the function works.

Reply:
The function is unclear, please add comments or refactor code:
1. What does incWin mean? Is it an abbreviation for something?
2. paramter "idx" is not a meaningful variable name, please rename and explain what the uses are.
3. Please explain for the other 2 similar functions "incWin" and "incDraw"

Reply:
This has been addressed.